### PR TITLE
fix repl from sbt

### PR DESCRIPTION
**Pull request by ravwojdyla**
After these fixes - scio shell can be started from both:
 * assembly jar via `java -jar <assembly-jar>`
 * sbt via `sbt run` or `sbt scio-repl/run`

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -128,6 +128,7 @@ val commonSettings = Project.defaultSettings ++ Sonatype.sonatypeSettings ++ ass
   }
 )
 
+lazy val printClasspath = taskKey[Unit]("Dump classpath")
```
Comment (nevillelyh) [inline build.sbt]: Is this necessary or debugging code?

Comment (ravwojdyla) [inline build.sbt]: Only for debugging purposes.

**Comment by ravwojdyla** [comment]
The seems to be some issue with `2.10` - let me double check.

**Comment by ravwojdyla** [comment]
Tests are fine and both `2.10` and `2.11` is fine for assembly jar. Sbt launch works with `2.11` but NOT with `2.10` - seems to be related changes between `2.10` and `2.11`. Not sure if it's worth to investigate `2.10` more (?)

**Comment by nevillelyh** [comment]
Not worth supporting 2.10 except the potential zeppelin use case but we're not quite there yet.
LGTM except the debugging lines. Remove them?

**Comment by bzz** [comment]
Sorry for digging this out, but have just came across this: just FYI, as of apache/zeppelin#747 Apache Zeppelin supports Scala 2.11 starting from upcoming 0.6.1 release.

**Comment by nevillelyh** [comment]
@bzz thanks that's good to know!

**Pull request merged**